server/configuration.go

```diff
@@ -209,8 +210,19 @@ func (p *Plugin) OnConfigurationChange() error {
 		return errors.Wrap(ensureBotError, "failed to ensure demo bot.")
 	}
 
+	config := p.API.GetConfig()
+	data, err := ioutil.ReadFile(*config.PluginSettings.Directory + "/com.mattermost.demo-plugin/assets/github.svg")
```

Comment:
I think it'd be better to use the `pluginID` as opposed to hardcoding it.

Reply:
You can just use https://developers.mattermost.com/extend/plugins/server/reference/#API.GetBundlePath instead.